### ISSUE TITLE
Import class fails when trying to autoload non-existing paths

### ISSUE THREAD

**Issue by Miljar** [phpactor/phpactor#419]
I have a project which uses PHPStan. In the `composer.json` of PHPStan, 2 paths are specified for the autoloader:

```json
"autoload": {
	"psr-4": {"PHPStan\\": ["src/", "build/PHPStan"]}
},
```
However, the `build/PHPStan` path doesn't exist when you install PHPStan via composer.

This causes PHPActor to fail when using the `import_class` RPC call (from Vim8):

<details>
  <summary>Response log</summary>
  <p>

```
[2018-04-17T09:45:51.408251+00:00] phpactor.ERROR: RESPONSE {"action":"error","parameters":{"message":"RecursiveDirectoryIterator::__construct(/Users/tom/Code/Dx/denp/argus-code/vendor/phpstan/phpstan/build/PHPStan): failed to open dir: No such file or directory","details":"0: RecursiveDirectoryIterator::__construct(/Users/tom/Code/Dx/denp/argus-code/vendor/phpstan/phpstan/build/PHPStan): failed to open dir: No such file or directory\n#0 /Users/tom/.vimfiles/plugged/phpactor/vendor/phpactor/source-code-filesystem/lib/Adapter/Composer/ComposerFileListProvider.php(81): RecursiveDirectoryIterator->__construct('/Users/tom/Code...')\n#1 /Users/tom/.vimfiles/plugged/phpactor/vendor/phpactor/source-code-filesystem/lib/Adapter/Composer/ComposerFileListProvider.php(62): Phpactor\\Filesystem\\Adapter\\Composer\\ComposerFileListProvider->createFileIterator(Object(Phpactor\\Filesystem\\Domain\\FilePath))\n#2 /Users/tom/.vimfiles/plugged/phpactor/vendor/phpactor/source-code-filesystem/lib/Domain/ChainFileListProvider.php(23): Phpactor\\Filesystem\\Adapter\\Composer\\ComposerFileListProvider->fileList()\n#3 /Users/tom/.vimfiles/plugged/phpactor/vendor/phpactor/source-code-filesystem/lib/Adapter/Simple/SimpleFilesystem.php(25): Phpactor\\Filesystem\\Domain\\ChainFileListProvider->fileList()\n#4 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/SourceCodeFilesystem/SourceCodeFilestem/Application/ClassSearch.php(40): Phpactor\\Filesystem\\Adapter\\Simple\\SimpleFilesystem->fileList('{ListDefinition...')\n#5 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/CodeTransform/Rpc/ImportClassHandler.php(141): Phpactor\\Extension\\SourceCodeFilesystem\\SourceCodeFilestem\\Application\\ClassSearch->classSearch('composer', 'ListDefinitionI...')\n#6 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/CodeTransform/Rpc/ImportClassHandler.php(72): Phpactor\\Extension\\CodeTransform\\Rpc\\ImportClassHandler->suggestions('ListDefinitionI...')\n#7 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/Rpc/RequestHandler/RequestHandler.php(39): Phpactor\\Extension\\CodeTransform\\Rpc\\ImportClassHandler->handle(Array)\n#8 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/Rpc/RequestHandler/ExceptionCatchingHandler.php(25): Phpactor\\Extension\\Rpc\\RequestHandler\\RequestHandler->handle(Object(Phpactor\\Extension\\Rpc\\Request))\n#9 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/Rpc/RequestHandler/LoggingHandler.php(39): Phpactor\\Extension\\Rpc\\RequestHandler\\ExceptionCatchingHandler->handle(Object(Phpactor\\Extension\\Rpc\\Request))\n#10 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/Rpc/Command/RpcCommand.php(74): Phpactor\\Extension\\Rpc\\RequestHandler\\LoggingHandler->handle(Object(Phpactor\\Extension\\Rpc\\Request))\n#11 /Users/tom/.vimfiles/plugged/phpactor/lib/Extension/Rpc/Command/RpcCommand.php(62): Phpactor\\Extension\\Rpc\\Command\\RpcCommand->processRequest(Object(Phpactor\\Extension\\Rpc\\Request))\n#12 /Users/tom/.vimfiles/plugged/phpactor/vendor/symfony/console/Command/Command.php(264): Phpactor\\Extension\\Rpc\\Command\\RpcCommand->execute(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#13 /Users/tom/.vimfiles/plugged/phpactor/vendor/symfony/console/Application.php(874): Symfony\\Component\\Console\\Command\\Command->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#14 /Users/tom/.vimfiles/plugged/phpactor/vendor/symfony/console/Application.php(228): Symfony\\Component\\Console\\Application->doRunCommand(Object(Phpactor\\Extension\\Rpc\\Command\\RpcCommand), Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#15 /Users/tom/.vimfiles/plugged/phpactor/lib/Application.php(42): Symfony\\Component\\Console\\Application->doRun(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#16 /Users/tom/.vimfiles/plugged/phpactor/vendor/symfony/console/Application.php(130): Phpactor\\Application->doRun(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#17 /Users/tom/.vimfiles/plugged/phpactor/bin/phpactor(42): Symfony\\Component\\Console\\Application->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#18 {main}"}} []
```
</p></details>

**Comment by dantleech** [comment]
This should have been fixed in develop yesterday #418 , can you confirm?

**Comment by Miljar** [comment]
Just tried it and I can confirm the fix. 👍
